astronomer/providers/google/cloud/sensors/gcs.py

```diff
@@ -39,32 +40,18 @@ class GCSObjectExistenceSensorAsync(BaseOperator):
         account from the list granting this role to the originating account (templated).
     """
 
-    template_fields = ("bucket", "object", "impersonation_chain")
```

Comment:
why are we removing `template_fields`

Reply:
Initially, it was inherited from `BaseOperator` now it's from `GCSObjectExistenceSensor` which contains this information already. So I have removed the `template_fields`.